integration_tests/tests/consistency/consistency_issue_enhanced.sql

```diff
@@ -5,7 +5,7 @@
 ) }}
 
 {# Exclude columns that depend on calculations involving the current time in seconds or aggregate strings in a random order, as they will differ between runs. #}
-{% set exclude_columns = ['open_duration_seconds', 'any_assignment_duration_seconds', 'last_assignment_duration_seconds'] %}
+{% set exclude_columns = ['open_duration_seconds', 'any_assignment_duration_seconds', 'last_assignment_duration_seconds', 'conversation', 'story_points', 'story_point_estimate'] %}
```

Comment:
We can remove these now to make sure they're included in future consistency tests.